packages/context/src/app/AppProvider.tsx

```diff
@@ -113,9 +114,16 @@ function AppProvider(props: AppProviderProps) {
                     const pageId = defaultPageId.includes('/')
                         ? defaultPageId.split('/').slice(2).join('/')
                         : defaultPageId;
-                    defaultPage = await apiServices.get<Page>(
-                        `/webContent/apps/${app.id}/pages/${encodeURIComponent(encodeURIComponent(pageId))}`,
-                    );
+                    try {
+                        defaultPage = await apiServices.get<Page>(
+                            `/webContent/apps/${app.id}/pages/${encodeURIComponent(encodeURIComponent(pageId))}`,
+                        );
+                    } catch (error) {
+                        const err = error as AxiosError;
+                        if (err.status === 404) {
```

Comment:
`AxiosError.status` is a string, and I'm not too sure what gets populated there.  I think it's safer to check the status on `AxiosError.response`.